tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+# Performance Harness Tests
+
+The `performance_test.py` script performs a binary search of EOS Token Transfers Per Second (TPS) range at configurable low test duration scoring each individual test scenario to find where TPS seems to be topping out. It does this by iteratively configuring and running `performance_test_basic.py` tests and analyzing the output to determine a success metric to continue the search.  The script then proceeds to conduct an additional search with longer duration test runs at a narrowed TPS window around the discovered maximum TPS throughput. Finally it produces a report on the entire performance run, summarizing each individual test scenario, results, and full report details on the tests when maximum TPS was achieved ([Performance Test Report](#performance-test))
+
+The `performance_test_basic.py` script performs a single basic performance test that targets a configurable TPS target and, if successful, reports statistics on perfomance metrics measured during the test.  It configures and launches a blockchain test environment, creates wallets and accounts for testing, configures and launches transaction generators for creating specific transaction load in the ecosystem.  Finally it analyzes the performance of the system under the configuration through log analysis and chain queries and produces a [Performance Test Basic Report](#performance-test-basic).
+
+The `launch_generators.py` script provides a means to easily calculate and spawn the number of transactions generator instances to generate a given target TPS, distributing generation load between the instances in a fair manner such that the aggregate load meets the requested test load.
+
+The `log_reader.py` script is used primarily to analyze `nodeos` log files to glean information about generated blocks and transactions within those blocks after a test has concluded.  This information is used to produce the performance test report. In similar fashion, `read_log_data.py` allows for recreating a report from the configuration and log files without needing to rerun the test.
+
+## Prerequisites
+
+Please refer to [Leap: Software Installation](https://github.com/AntelopeIO/leap#software-installation)
+
+## Steps
+
+1. Install Leap. For complete instructions on obtaining compiled binaries or building from source please refer to [Leap: Building From Source](https://github.com/AntelopeIO/leap#building-from-source)
```

Comment:
This instruction and the previous instruction are a little contradictory.  While they point to the same page, the link titles make them appear distinct.  We should unambiguously recommend building from source until the binary packages are either revised to include these scripts or a supplemental package containing them is available.

Reply:
The two links are distinct in that they point to different sections on the same page.

Reply:
I think I just got what you were saying.  Will do.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)